src/stylesheets/app.scss

```diff
@@ -1,115 +1,155 @@
 @import "base/typography";
 @import url('https://fonts.googleapis.com/css?family=Montserrat:300,300i,400,400i');
-* {
-    box-sizing: border-box;
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
 }
 
 body {
-    background-color: #DEEDF0;
+  background-color: #DEEDF0;
 }
 
 .container {
-    font-family: 'Montserrat', sans-serif;
-    margin: 0 auto;
-    width: 480px;
+  font-family: 'Montserrat', sans-serif;
+  margin: 0 auto;
+  width: 480px;
 }
 
 header {
-    background-color: #a15693;
-    border-top-left-radius: 15px;
-    border-top-right-radius: 15px;
-    padding: 0.9375rem 1.875rem 0.6875rem 1.875rem;
-    position: relative;
-    p {
-        color: white;
-        font-size: 1.5rem;
-        font-weight: 400;
-        margin: 0;
-        text-transform: uppercase;
-        width: 87.5%;
-    }
-    img {
-        position: absolute;
-        right: 21px;
-        top: 22px;
-    }
+  background-color: #a15693;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+  padding: 0.9375rem 1.875rem 0.6875rem 1.875rem;
```

Comment:
Can't say I've ever seen `rem`s used for padding. Innnnteresting.  (I prefer pixels for spacing myself, since it's easier to make them consistent and match up)